tn_api_pricing_booking_spec_nilou.yml

```diff
@@ -603,6 +607,11 @@ components:
             example: USD
             required: true
             type: string
+        region:
```

Comment:
Why do we have region at root level and at the credential level? I think it should be cred level only

Reply:
oops right in code it's not in credentials, I'll add it there and remove it from root